src/meta/src/rpc/ddl_controller_v2.rs

```diff
@@ -46,7 +46,7 @@ impl DdlController {
         let job_id = streaming_job.id();
 
         match &mut streaming_job {
-            StreamingJob::Table(Some(src), table, job_type) => {
+            StreamingJob::Table(src, table, job_type) => {
                 // If we're creating a table with connector, we should additionally fill its ID first.
```

Comment:
Just curious: why v2 doesn't generate id for the `Source`?

Reply:
It's already filled in catalog controller, you can check function `create_job_catalog` for more details.